README.adoc

```diff
@@ -25,6 +25,42 @@ Site content for the Ansible Bootcamp Enablement.
     └── example_partial.adoc
 ----
 
+=== Using ETX-Ansible DevSpaces
+
+Once you have a lab environment set up, you can use DevSpaces to develop additional content. To do this you should first access your devspace environment by clicking the 9 square icon in the top right of the console and clicking, "Red Hat OpenShift DevSpaces".
+
+If it is your first time using this cluster's devspaces you will need to authorize oauth for DevSpaces. Then in the top right drop down next to your username and icon and select, "User Preferences". In the User Preferences page you will want to select "Personal Access Tokens" and add a Github Accerss token. See creating a Github Personal Access Token section below.
```

Comment:
Do we need GitHub? We are using Gitea for everything in this bootcamp where possible

Reply:
It is to update this repo's content.